script/restore.sh

```diff
@@ -1,29 +1,47 @@
-#!/bin/bash -e
-BDIR=$(readlink -f "$1")
```

Comment:
Having to deal with PTY I/O is a pain for `foreman-admin`, not to mention it breaks `1.8.x` compatibility. May i suggest instead the user having to pass a `--confirm` or something to similar to make sure they aren't just running the script willy-nilly without understanding the consequences?